src/components/AppProvider.tsx

```diff
@@ -26,42 +29,18 @@ const config: SDKConfig = {
   errorComponent: MetabaseError,
 }
 
-const theme: MetabaseTheme = {
-  fontFamily: "Custom",
-  fontSize: "14px",
-  colors: {
-    brand: "#FF8000",
-    filter: "#00D9CC",
-    "text-primary": "#F8F7F7",
-    "text-secondary": "#F8F7F7",
-    "text-tertiary": "#F8F7F7",
-    border: "#4C4A48",
-    background: "#212121",
-    "background-hover": "#4C4A48",
-    charts: ["#00D9CC"],
-    positive: "#4AC40E",
-    negative: "#FF0F00",
-  },
-  components: {
-    cartesian: {
-      padding: "4px 10px",
-    },
-    dashboard: {
-      card: {
-        border: "1px solid #4C4A48",
-      },
-    },
-    scalar: {
-      value: {
-        fontSize: "47px",
-        lineHeight: "50px",
-      },
-    },
-  },
-}
+export const AppProvider = ({ children }: Props) => {
+  const [siteKey] = useAtom(siteAtom)
```

Comment:
I found we name this variable from `siteAtom` 3 ways
- `siteKey`
- `siteName`
- `theme`

If you like the explicitness I'd choose either of the first two, but I think `site` is also enough.

Can we ensure we name all of these variables the same? Or do you think it it's valid to name these differently based on where the atim is used?

Reply:
renamed - thanks for the keen eye!